docs/docs/howto/authentication/azure-ad.rst

```diff
@@ -0,0 +1,56 @@
+Azure AD
+========
+
+Users can login with their credentials configured in Azure Active directory. On the login screen user, is prompted with a login option for Azure AD.
+
+After clicking on the login button, user is provided with an Azure pop up window with options to provide his/her credentials or select an Azure account.
+
+* Configuration
+
+1. Make sure Klaw is running in secure mode::
```

Comment:
I think mentioning that this is all configured in the application.properties of klaw-core would help. 
It is mentioned in point 4 but i think moving it up would be great.